app/assets/javascripts/components/profile_cards.es6.jsx

```diff
@@ -0,0 +1,23 @@
+class ProfileCards extends Component {
```

Comment:
@sonia-y this is the parent component of `ProfilePreview` and `ProfileOptions` I was talking about!